docs/workflows/alternate-content-source.rst

```diff
@@ -0,0 +1,46 @@
+Alternate Content Sources
```

Comment:
There is no information here about what alternate content sources actually are, or why you would want to use them.  It would be good to add some details about this, or if it's documented elsewhere, a link to that documentation.

Reply:
Added short description.

Reply:
I'd also link to the docs for ACS in core.